code/starfield/starfield.cpp

```diff
@@ -151,25 +150,10 @@ typedef struct vDist {
 
 star Stars[MAX_STARS];
 
-motion_debris Motion_debris[MAX_DEBRIS];
+motion_debris_instance Motion_debris[MAX_MOTION_DEBRIS];
 
-
-typedef struct debris_vclip {
-	int	bm;
-	int	nframes;
-	char  name[MAX_FILENAME_LEN];
-} debris_vclip;
-extern debris_vclip Debris_vclips_normal[];
-extern debris_vclip Debris_vclips_nebula[];
-extern debris_vclip *Debris_vclips;
-
-//XSTR:OFF
-debris_vclip Debris_vclips_normal[MAX_DEBRIS_VCLIPS] = { { -1, -1, "debris01" }, { -1, -1, "debris02" }, { -1, -1, "debris03" }, { -1, -1, "debris04" } };
-debris_vclip Debris_vclips_nebula[MAX_DEBRIS_VCLIPS] = { { -1, -1, "nebdeb01" }, { -1, -1, "nebdeb02" }, { -1, -1, "nebdeb03" }, { -1, -1, "nebdeb04" } };
-debris_vclip *Debris_vclips = Debris_vclips_normal;
-//XSTR:ON
-
-int stars_debris_loaded = 0;	// 0 = not loaded, 1 = normal vclips, 2 = nebula vclips
+SCP_vector<motion_debris_types> Motion_debris_info;
+motion_debris_bitmaps* Motion_debris_ptr;
```

Comment:
This should be initialized to `nullptr` here.  There's only one other place in the code it is set to `nullptr`, and there are ways it could be skipped.

Reply:
done